plugins/SLURM-Integration/ufm_slurm_prolog.py

```diff
@@ -45,9 +45,14 @@ def prolog_init(self):
                         Constants.TOKEN_AUTH, Constants.CONF_TOKEN))
                     sys.exit(self.should_fail)
                 self.session = self.ufm.getServerSession(auth_type=self.auth_type, token=token)
+            elif self.auth_type == Constants.KERBEROS_AUTH:
+                principal_name = self.general_utils.get_conf_parameter_value(Constants.CONF_PRINCIPAL_NAME)
+                principal_pass = self.general_utils.get_conf_parameter_value(Constants.CONF_PRINCIPAL_PASSWORD)
+                self.session = self.ufm.getServerSession(auth_type=self.auth_type, principal_name=principal_name,
+                                                         principal_password=principal_pass if principal_pass else "")
             else:
-                logging.error("auth_type in ufm_slurm.conf file must be one of the following (%s, %s)" % (
-                    Constants.BASIC_AUTH, Constants.TOKEN_AUTH))
+                logging.error("auth_type in ufm_slurm.conf file must be one of the following (%s, %s, %s)" % (
```

Comment:
duplicate error message from ufm_slurm_epilog - please add some string which identify the error message source (epilog / prolog)

Reply:
it's clear in the logs file from which script the logs are coming.